cmip6_downscaling/methods/common/tasks.py

```diff
@@ -85,20 +92,43 @@ def get_experiment(run_parameters: RunParameters) -> UPath:
 
 
 @task
-def rechunk(path: UPath, pattern: str = None) -> UPath:
+def rechunk(path: UPath, pattern: str, run_parameters: RunParameters) -> UPath:
+    target = (
+        intermediate_dir
+        / "rechunked_stores_"
```

Comment:
```suggestion
        / "rechunk"
```

As a pattern, I've just been using the task name for this part of the path